operations/reservations.md

```diff
@@ -529,7 +529,8 @@ Adds the specified reservations as a single group. If `GroupId` is specified, ad
                 {
                     "ProductId": "3dc5d79b-67ce-48ed-9238-47fcf5d1a59f"
                 }
-            ]
+            ],
+            "BookerId": "b22bf671-ccdf-40aa-a7e6-b20a4f91d79a"
```

Comment:
@vitasamek @MartinPrazak  Why isn't this close to CustomerId? Ordering is important as well, both here and in the table. Do not add all new things on the bottom, that way you end up with completely random ordering.